tests/acceptance/WPFirstCest.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+
+class WPFirstCest
```

Comment:
I see. I know it's hard because we don't have any real testable features yet. But I'd try to make it a real -- or realistic -- test. Something like:

```
# tests/acceptance/LoginTest.php
class UserLoginTest {
  // Tests
  public function expectsUserToLogin(AcceptanceTester $I) {
  }

  public function expectsAdminToLogin(AcceptanceTester $I) {
  }
}
```

Is the `WP` prefix necessary? I find it redundant. It looks like Codeception likes to suffix their text with `Cept`. I don't see the point with that either.